downloads/swan-lake-release-notes/swan-lake-preview8/RELEASE_NOTE.md

```diff
@@ -0,0 +1,519 @@
+---
+layout: ballerina-blank-page
+title: Release Note
+---
+
+### Overview of Ballerina Swan Lake Preview 8 
+
+This release is the eighth preview version of Ballerina Swan Lake. It includes a new set of language features along with improvements and bug fixes to the compiler, runtime, standard library, and developer tooling.
+
+- [Updating Ballerina](#updating-ballerina)
+    - [For Existing Users](#for-existing-users)
+    - [For New Users](#for-new-users)
+- [Highlights](#highlights)
+- [What is new in Ballerina Swan Lake Preview 8](#what-is-new-in-ballerina-swan-lake-preview-8)
+    - [Language](#language)
+        - [Support Identifier Escapes Without an Initial Quote](#support-identifier-escapes-without-an-initial-quote)
+        - [Included Record Parameters](#included-record-parameters)
+        - [Service Typing Changes](#service-typing-changes)
+        - [Transactional Services](#transactional-services)
+    - [Standard Library](#standard-library)
+        - [HTTP Module Changes](#http-module-changes)
+        - [Log Module Changes](#log-module-changes)
+        - [Email Module Changes](#email-module-changes)
+        - [WebSub Module Changes](#websub-module-changes)
+        - [UUID Module Changes](#uuid-module-changes)
+        - [Introduced New Modules](#introduced-new-modules)
+
+#### Updating Ballerina
+
+You can use the update tool to update to Ballerina Swan Lake Preview 8 as follows.
+
+##### For Existing Users
+
+If you are already using Ballerina, you can directly update your distribution to the Swan Lake channel using the [Ballerina Update Tool](http://ballerina.io/swan-lake/learn/keeping-ballerina-up-to-date/). To do this, first, execute the command below to get the update tool updated to its latest version. 
+                        
+> `ballerina update`
+
+ Next, execute the command below to update to Swan Lake Preview 8.
+
+ > `ballerina dist pull slp8`                 
+
+However, if you are using a Ballerina version below 1.1.0, install via the [installers](https://ballerina.io/downloads/).
+
+##### For New Users
+
+If you have not installed Ballerina, then download the [installers](https://ballerina.io/downloads/) to install.
+
+#### Highlights
+
+- Ability to use `\` to escape the special characters in an identifier without mentioning the initial quote
+- Ability to specify included record parameters
+- Introduction of service typing changes basing services on objects
+- Extension of Ballerina transaction capabilities to define transactional resource functions and transactional remote functions
+- Improvements to the HTTP, Log, Email, WebSub, and UUID Standard Library modules
```

Comment:
UUID is a new module. So shouldn't it come in the next bullet point? @daneshk , @MadhukaHarith92 ?

Reply:
Should come under `##### Introduced New Modules` .

Reply:
Fixed.